webapp/views/partials/users.jade

```diff
@@ -1,4 +1,5 @@
 div(class='container', ng-if='init()')
+    -var selected = 'Users';
```

Comment:
Why is there a - before var? What is this variable even used for? Shouldn't it be declared in the controller instead?